v3-docs/v3-migration-docs/breaking-changes/index.md

```diff
@@ -0,0 +1,207 @@
+# Breaking changes
+
+## MongoDB Methods in the server
+
+As mentioned in the [overview](../index.md#mongo-methods-server) `insert`, `update`,
+ `remove`, `find`, `findOne`, `upsert` methods no longer work in the server.
+
+You should migrate to use their `Async` counterparts.
+
+
+```js
+const docs = MyCollection.find({ _id: '123' }).fetch(); // [!code error] This will not work in the server
+const doc = MyCollection.findOne({ _id: '123' }); // [!code error] This will not work in the server
+
+
+// in Meteor 3.x you should use the Async methods
+
+const docs = await MyCollection.find({ _id: '123' }).fetchAsync(); // [!code highlight] This will work in the server
+const doc = await MyCollection.findOneAsync({ _id: '123' }); // [!code highlight] This will work in the server
+
+```
+
+## CLI
+
```

Comment:
Instead of directing users to other links, let's explain here the reason why Vue 2 is no longer supported.